sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinitionTest.java

```diff
@@ -54,34 +54,18 @@ public void test() {
       "owaspTop10-2021:a9");
   }
 
-  @Test
-  public void test_before_9_3() {
-    assertThat(getSecurityStandards(Version.create(9, 2), "S1111")).containsExactlyInAnyOrder("cwe:117", "cwe:532", "owaspTop10:a10", "owaspTop10:a3");
-  }
-
-  @Test
-  public void test_security_standards_9_4_PCI_DSS_is_not_available() {
-    assertThat(getSecurityStandards(Version.create(9, 4), PCI_DSS_RULE_KEY)).isEmpty();
-  }
-
   @Test
```

Comment:
What about this test?
Seems to be testing version 9.5.

Reply:
The version number passed here is not used in our Java codebase, it's just passed to the Plugin API. So there's not much we can do about it. I removed the test for version 9.4, because the minimum API version is 9.9, but this test should still work. I could change the version number to 9.9, but it wouldn't matter; at least it shows from which version this feature is available.
The same is true for other occurrences I didn't remove: the runtime versions are passed to the API, and you can only see them in the code from the decompiled .class files. We cannot remove them.